### PR TITLE
implement_methods fixes

### PR DESCRIPTION
**Pull request by marcellourbani**
I think it's working ok, not too sure about the fix proposal.

Also, as this is about unknown interfaces, it will emit false positives when the target is not a method.

### CODE REVIEW

File: packages/core/test/rules/implement_methods.ts [hunk 1]
```diff
@@ -523,4 +523,73 @@ ENDCLASS.`;
...
+        contents: `CLASS lcl DEFINITION.
+    PUBLIC SECTION.
+      ALIASES blah FOR zlif~blah.
+  ENDCLASS.
+  
+  CLASS lcl IMPLEMENTATION.
+  ENDCLASS.`,
+      },
+    ]);
+    expect(issues.length).to.equals(1);
```
Comment (larshp) [inline packages/core/test/rules/implement_methods.ts:539]: hmm, should this be an error? `blah` might not be a method

Comment (marcellourbani) [reply packages/core/test/rules/implement_methods.ts:539]: That's what I meant above. BTW, I thought I might have to check parent classes too to get this right

**Comment by larshp** [comment]
🤷‍♂️ lets just merge it

**Comment by larshp** [comment]
need to roll it back, due to regression,

[ABAP-Logger/src/zcl_logger.clas.abap:156](https://github.com/ABAP-Logger/ABAP-Logger/blob/main/src/zcl_logger.clas.abap#L156): Implement method "ty_symsg" from interface "zif_loggable_object"(implement_methods)
[ABAP-Logger/src/zcl_logger.clas.abap:156](https://github.com/ABAP-Logger/ABAP-Logger/blob/main/src/zcl_logger.clas.abap#L156): Implement method "ty_message" from interface "zif_loggable_object"(implement_methods)
[ABAP-Logger/src/zcl_logger.clas.abap:156](https://github.com/ABAP-Logger/ABAP-Logger/blob/main/src/zcl_logger.clas.abap#L156): Implement method "tty_messages" from interface "zif_loggable_object"(implement_methods)

**Comment by marcellourbani** [comment]
Will look into it

**Pull request merged**